# Building a native, statically-linked insmod can result in ld failing because
# it can't find a static libc.  A native, static insmod is unnecessary anyway,
# so this patch just removes it.
#
# - jdike@linux.intel.com

Index: module-init-tools-3.2.2/Makefile.am
===================================================================
--- module-init-tools-3.2.2.orig/Makefile.am	2010-05-26 15:43:16.000000000 -0400
+++ module-init-tools-3.2.2/Makefile.am	2010-05-26 16:04:06.232990785 -0400
@@ -5,16 +5,10 @@ rmmod_SOURCES = rmmod.c testing.h
 depmod_SOURCES = depmod.c moduleops.c tables.c zlibsupport.c depmod.h moduleops.h tables.h list.h testing.h  zlibsupport.h
 modinfo_SOURCES = modinfo.c zlibsupport.c testing.h zlibsupport.h
 
-insmod_static_SOURCES = insmod.c
-insmod_static_LDFLAGS = -static
-# We don't want the $(zlib_flags) here: that makes a dynamic executable
-insmod_static_LDADD = 
-
 EXTRA_insmod_SOURCES = backwards_compat.c
 EXTRA_lsmod_SOURCES = backwards_compat.c
 EXTRA_modprobe_SOURCES = backwards_compat.c
 EXTRA_rmmod_SOURCES = backwards_compat.c
-EXTRA_insmod_static_SOURCES = backwards_compat.c
 EXTRA_depmod_SOURCES = moduleops_core.c
 EXTRA_modinfo_SOURCES = backwards_compat.c
 
